iosHackthonApp/.swiftlint.yml

```diff
@@ -0,0 +1,21 @@
+identifier_name:
+  min_length: 2
```

Comment:
This allows us to use things like `dy` or `dx` for coordinates and a string named: `ok`

Reply:
Nice "id" is another common one that swiftLint doesn't like out of the box!

Reply:
Do we need to add a `max_length`?

Reply:
Nothing so far is triggering any warning so I've left it as the default